<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_libro_iva_compras</class>
 <widget class="QWidget" name="form_libro_iva_compras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Libro IVA Compras</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Liberation Sans&quot;;
color: rgb(0, 0, 0);
background-color: rgb(156, 156, 156);
selection-background-color: rgb(164, 190, 221);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>CUIT / CUIL: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_cuit">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Razón Social: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_razon_social">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btn_buscar">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(119, 149, 177)</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ejercicio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbx_ejercicio">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_buscar_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(119, 149, 177)</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbx_mes">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>Enero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Febrero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Marzo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Abril</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mayo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Junio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Julio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agosto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Septiembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Octubre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Noviembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Diciembre</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget{
background-color: rgb(123, 121, 143);
}
QLineEdit {
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QComboBox{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Libro IVA Compras </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tw_compras">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Liberation Sans&quot;;</string>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <column>
          <property name="text">
           <string>Fecha</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>N° Comprobante</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ref</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Proveedor</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cond. Fte. IVA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CUIT</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>10,5%</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>21%</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>27%</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MONOTRIBUTO</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>No Grav./Op Exentas</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IVA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Percep. IIBB</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Imp. Gasoil</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Percep. IVA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TOTAL</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Resultado: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_resultado_uno">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_resultado_dos">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_eliminar">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(119, 149, 177)</string>
           </property>
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="btn_modificar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(119, 149, 177)</string>
           </property>
           <property name="text">
            <string>Actualizar</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="btn_nuevo">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(119, 149, 177)</string>
           </property>
           <property name="text">
            <string>Nuevo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lne_cuit</tabstop>
  <tabstop>lne_razon_social</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>cbx_ejercicio</tabstop>
  <tabstop>btn_buscar_2</tabstop>
  <tabstop>cbx_mes</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tw_compras</tabstop>
  <tabstop>lne_resultado_uno</tabstop>
  <tabstop>lne_resultado_dos</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>btn_modificar</tabstop>
  <tabstop>btn_nuevo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
